tensorflow/stream_executor/dnn.h

```diff
@@ -446,6 +446,14 @@ class FilterDescriptor {
     return AsInt64Slice(tensor_.dimensions()).subspan(2);
   }
 
+  // Full dimensions of the underlying filter,
+  // ordered according to a specific layout.
+  std::vector<int64> full_dims(const FilterLayout& layout) const;
+
+  // Full strides of the underlying filter,
+  // ordered according to a specific layout.
+  std::vector<int64> full_strides(const FilterLayout& layout) const;
```

Comment:
I'd prefer a comment of "ROCm: copied from DataLayout type implementations". If the `DataLayout type`'s implementation is to be changed sometime in the future, we may want to sync their changes to ours, or vice versa.

Reply:
don't think I follow you here...can you elaborate?

Reply:
I was referring to those lines: 
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/d2ac4d66ce7b5672ecf76ecafe55a35f483464e3/tensorflow/stream_executor/dnn.h#L262-L268

Just so that it give a pointer to future developers that those two pieces of implementation should keep in sync.

Reply:
now I see what you mean....will add those comments in, when I file the corresponding upstream PR